docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,63 @@
+.NET Core
+=========
+
+.NET Core is a general purpose development platform developed by Microsoft and the .NET community on [GitHub](https://github.com/dotnet/core). It is cross-platform, supports Windows, OS X and Linux and intends to support a wide variety of CPUs and OSes over time. It can be used in device, cloud and embedded/IoT scenarios. We expect it to be extended to support additional scenarios over time (e.g. WebAssembly), both by Microsoft and the community.
+
+.NET Core is composed of a few parts:
+
+- A [.NET runtime](https://github.com/dotnet/coreclr) that can be used for a wide variety of app types
+- A set of [framework libraries]((https://github.com/dotnet/corefx)). 
+- A set of tools and compilers that enable the base developer experience, available in the [.NET Core SDK]((https://github.com/dotnet/cli)).
+
+The C#, F# and VB compilers all support targeting and running on .NET Core. .NET Core tools are integrated into severals text editors and IDEs, including Visual Studio, Visual Studio Code, Sublime Text and Vim, providing .NET Core development experiences in your favorite environment. 
+
+.NET Core is open source and was contributed to the [.NET Foundation](http://dotnetfoundation.org) by Microsoft in 2014 and is now one of the most active .NET Foundation projects. It can be freely adopted by individuals and companies, including for personal, academic or commercial purposes. Multiple companies are using .NET Core as part of apps, tools, new platforms and hosting services. Some of these companies make significant contributions to .NET Core on GitHub and provide guidance on the product direction as part of the [.NET Foundation Technical Steering Group](http://www.dotnetfoundation.org/blog/tsg-welcome).
+
+.NET Core v1 supports the following workloads and app-models:
+
+- Class Libraries
+- Console apps
+- ASP.NET Core websites and services
+- UWP Windows 10 apps
+
+You can get started with .NET Core in just a few minutes:
+
+- [Getting started with .NET Core](https://aka.ms/dotnetcoregs)
+- [.NET Core samples](https://github.com/dotnet/core)
+- [.NET Core OS and CPU Support](link)
+- [Explore .NET Core repos](https://github.com/dotnet/core)
+
+Composition of .NET Core
+------------------------
+
+**.NET Core is the following:**
+
+- **A set of packages** -- These contain the runtime and the API shape and implementation of .NET Core.
+- **A set of metapackages (e.g. .NET Standard Library)** --  They describe various layers of .NET Core by referencing the appropriate set of library packages.
+- **A [distribution](https://aka.ms/dotnetcoregs) of .NET Core from Microsoft** -- This includes the CoreCLR runtime, associated libraries, the console application host and the `dotnet` launcher. It is described by the Microsoft.NETCore.App metapackage.
```

Comment:
Nit: include backticks `` around the package ids. We use that around for code-related stuff.